### ISSUE TITLE
Parser error

### ISSUE THREAD

**Issue by gquerret** [Riverside-Software/sonar-openedge#915]
On behalf of VIF

CABL Version 2.14.3, not tested on previous versions. This code block generates a parser issue:
```
def var rct1 as widget-handle.
assign rct1:height = .1
       rct1:row  = 5.71.
```

Message:
```
org.antlr.v4.runtime.misc.ParseCancellationException: Mismatched input ''=.1rct1'', expecting '{DOUBLECOLON, EQUAL, LEFTBRACE, OBJCOLON, STAREQUAL, SLASHEQUAL, PLUSEQUAL, MINUSEQUAL, IN}'
```

**Comment by gquerret** [comment]
Problem identified in commit cf5439e

**Comment by gquerret** [comment]
Fixed, will be available in 2.14.4.
